### PR TITLE
Issue 43: Patch changed data set name in README

### PR DESCRIPTION
**Pull request by seabbs**
This PR closes issue #43 by renaming the dataset to the new dataset name. 

Looking at the README do we really need to list contributor qualifications after their names and if so why aren't we listing more than just PhDs. I would much rather drop.

### CODE REVIEW

**Comment by zsusswein** [comment]
> do we really need to list contributor qualifications after their names and if so why aren't we listing more than just PhDs. I would much rather drop.

Fine by me. Would you mind dropping them here?

**Comment by seabbs** [comment]
Happy to do so but I thought it was a Predict wide thing

File: README.md [hunk 1]
```diff
@@ -39,18 +39,18 @@ This snippet uses the simulated dataset to demonstrate how to use the main model
...
        )
 print(fit)
 ```
 
 ## Project Admin
 
 - Zachary Susswein (@zsusswein)
-- Katelyn Gostic, PhD (@kgostic)
-- Sam Abbott, PhD (@seabbs)
+- Katelyn Gostic (@gostic)
```
Comment (zsusswein) [inline README.md:52]:
```suggestion
- Katelyn Gostic (@kgostic)
```

**Review by zsusswein — APPROVED** [review]

**Comment by seabbs** [comment]
Do you want a news update or happy as is?

**Comment by zsusswein** [comment]
Nope fine as is.

**Pull request merged**